frontend/src/components/modals/NewCredentialIssuerDialog.vue

```diff
@@ -124,11 +138,14 @@ export default {
   mixins: [ModalMixin],
   data() {
     return {
+      image: "",
+      remoteUrl: "",
```

Comment:
Does `image` and `remoteUrl` used somewhere?

Reply:
removed